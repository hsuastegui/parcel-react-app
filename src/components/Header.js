import React from "react";
import "./header.css"; // eslint-disable-line import/no-unassigned-import

const Header = () => {
  return <header>Welcome</header>;
};

export default Header;
