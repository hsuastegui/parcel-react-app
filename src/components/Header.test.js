import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import Header from "./Header";

describe("<Header />", () => {
  test("it works", () => {
    const wrapper = shallow(<Header />);

    expect(wrapper.find("header")).toHaveLength(1);
    expect(wrapper.find("header").text()).toEqual("Welcome");
  });
  test("it matches snapshot", () => {
    const tree = renderer.create(<Header />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
