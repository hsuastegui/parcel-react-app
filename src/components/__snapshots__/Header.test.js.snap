// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> it matches snapshot 1`] = `
<header>
  Welcome
</header>
`;
